examples/tests/backend_driver_sgskip.py

```diff
@@ -6,339 +6,48 @@
 This is used to drive many of the examples across the backends, for
 regression testing, and comparing backend efficiency.
 
-You can specify the backends to be tested either via the --backends
-switch, which takes a comma-separated list, or as separate arguments,
-e.g.
+You can specify the backends to be tested via the --backends
+switch, which takes a space-separated list, e.g.
 
-    python backend_driver.py agg ps
+    python backend_driver.py --backends agg ps
```

Comment:
this name needs to be updated

Reply:
I'm confused, which name?

Reply:
I just pushed a fix (the name of the file it self!)